include/mbedtls/config_adjust_legacy_from_psa.h

```diff
@@ -344,7 +344,6 @@
     defined(MBEDTLS_PSA_ECC_ACCEL_INCOMPLETE_KEYTYPES_BASIC)
 #define MBEDTLS_PSA_BUILTIN_PAKE 1
 #define MBEDTLS_PSA_BUILTIN_ALG_JPAKE 1
-#define MBEDTLS_ECP_DP_SECP256R1_ENABLED
```

Comment:
The PSA wrapper code hard-codes secp256r1 and SHA-256. It doesn't work with other algorithms. So we do need `MBEDTLS_ECP_DP_SECP256R1_ENABLED`, or more precisely `PSA_WANT_ALG_JPAKE` won't do anything useful without it and `MBEDTLS_MD_CAN_SHA256`.

Reply:
Right, I completely misremembered how our current PSA implementation is tied to these algs, sorry.